servers/rendering_server.cpp

```diff
@@ -1895,6 +1895,10 @@ void RenderingServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("has_os_feature", "feature"), &RenderingServer::has_os_feature);
 	ClassDB::bind_method(D_METHOD("set_debug_generate_wireframes", "generate"), &RenderingServer::set_debug_generate_wireframes);
 
+	ClassDB::bind_method(D_METHOD("is_render_loop_enabled"), &RenderingServer::is_render_loop_enabled);
+	ClassDB::bind_method(D_METHOD("set_render_loop_enabled", "render_loop_enabled"), &RenderingServer::set_render_loop_enabled);
```

Comment:
Make the argument only `enable`, otherwise it's too redundant.

Reply:
Ah yeah sorry, forgot about that binding as well

Reply:
Fixed